packages/utilities/src/AutoScroll.ts

```diff
@@ -23,6 +23,7 @@ export class AutoScroll {
   private _scrollableParent: HTMLElement | null;
   private _scrollRect: IRectangle | undefined;
   private _scrollVelocity: number;
+  private _scrollDirection: boolean;
```

Comment:
Nit: This seems to represent vertical scroll, but it's not apparent from naming. Maybe `_scrollVertical` would be more descriptive?

Reply:
Sorry, I was heads down on something but see that you changed it to "_isVerticalScroll" which is perfect, thanks @JasonGore and @dzearing !